### ISSUE TITLE
CMS - change username display in listing pages

### ISSUE THREAD

**Issue by troughc** [culturecreates/footlight-app#848]
Remove underline from usernames in the list. Keep them bold. Like this:

![Image](https://github.com/culturecreates/footlight-app/assets/64489312/fac4408c-7158-4bdd-b8c5-a6b13bf6ec36)

**Comment by troughc** [comment]
@sahalali @saumier  this issue was closed and completed BEFORE I reviewed it. Can you please check the process? 
I have now tested and reviewed: PASSED.

**Comment by AbhishekPAnil** [comment]
> @sahalali @saumier  this issue was closed and completed BEFORE I reviewed it. Can you please check the process? 
> I have now tested and reviewed: PASSED.

@troughc @SuhailAliyar @saumier This happened because , the close tag was added to the commit that was already in develop.@troughc Can we  reopen this and add it to review?

**Comment by troughc** [comment]
I don't think we need to reopen it @AbhishekPAnil I have tested and PASSED the issue. 
thanks!
